com.unity.graphtools.foundation/Editor/UI/Views/GraphView/GraphView.cs

```diff
@@ -1631,12 +1631,11 @@ protected virtual void DeleteElementsFromChangeSet(GraphModelStateComponent.Chan
 
                 k_UpdateAllUIs.Clear();
 
-                // GTF-Edit: Commented this out because on deletion it tries to represent our port models and fails
                 // ToList is needed to bake the dependencies.
-                //foreach (var ui in model.GetDependencies().ToList())
-                //{
-                //    ui.UpdateFromModel();
-                //}
+                foreach (var ui in model.GetDependencies().ToList())
```

Comment:
this is a GTF-Edit that's not needed, and also wasn't added to the PR we just submitted